pkg/jobrunaggregator/jobrunaggregatorlib/big_query.go

```diff
@@ -92,8 +94,34 @@ func (d dryRunInserter) Put(ctx context.Context, src interface{}) (err error) {
 	buf := &bytes.Buffer{}
 	fmt.Fprintf(buf, "BULK INSERT into %v\n", d.table)
 	for i := 0; i < srcVal.Len(); i++ {
-		s := srcVal.Index(i).Interface()
-		fmt.Fprintf(buf, "\tINSERT into %v: %v\n", d.table, s)
+
+		switch srcVal.Index(i).Interface().(type) {
```

Comment:
you should be able to assign it up here so you don't have to type assert in every case below.

`switch t := foo` I think.

Reply:
ah nice -- thanks!